packages/workers-shared/router-worker/src/index.ts

```diff
@@ -75,6 +77,69 @@ export default {
 
 			const maybeSecondRequest = request.clone();
 
+			if (config.static_routing) {
+				// evaluate "exclude" rules
+				const excludeRulesMatcher = generateStaticRoutingRuleMatcher(
+					config.static_routing.exclude ?? []
+				);
+				if (
+					excludeRulesMatcher({
+						request,
+					})
+				) {
+					// direct to asset worker
+					analytics.setData({
+						dispatchtype: DISPATCH_TYPE.WORKER,
+						staticRoutingDecision: STATIC_ROUTING_DECISION.EXCLUDE,
+					});
+					return await env.JAEGER.enterSpan("dispatch_worker", async (span) => {
```

Comment:
```suggestion
					return await env.JAEGER.enterSpan("dispatch_assets", async (span) => {
```

I'm not sure what the purpose of this + `dispatchType` tag is, but whatever, it seems to be how we're doing asset dispatches down below anyway.

Reply:
i agree - the span name and the label are redundant, but yeah i was just mirroring existing behavior